modules/migration-dvm-error-node-selectors.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+// * migration/migrating_3_4/troubleshooting-3-4.adoc
+// * migration/migrating_4_1_4/troubleshooting-4-1-4.adoc
+// * migration/migrating_4_2_4/troubleshooting-4-2-4.adoc
+
+[id="migration-dvm-error-node-selectors_{context}"]
+= Direct volume migration does not complete
+
+If direct volume migration does not complete, the target cluster might not have the same `node-selector` annotations as the source cluster.
+
+{mtc-short} migrates namespaces with all annotations in order to preserve security context constraints and scheduling requirements. During direct volume migration, {mtc-short} creates Rsync transfer pods on the target cluster in the namespaces that were migrated from the source cluster. If a target cluster namespace does not have the same annotations as the source cluster namespace, the Rsync transfer pods cannot be scheduled. They remain in a `Pending` state.
```

Comment:
Should `{mtc-short}` here be `{mtc-full} ({mtc-short})`? I noticed some of the modules in this assembly started off spelling it out for the first instance, but not all.

Reply:
s/in order to/to/

Reply:
s/They/The pods/